src/lang/english.txt

```diff
@@ -1314,7 +1314,7 @@ STR_CONFIG_SETTING_SHOW_NEWGRF_NAME                             :Show the NewGRF
 STR_CONFIG_SETTING_SHOW_NEWGRF_NAME_HELPTEXT                    :Add a line to the build vehicle window, showing which NewGRF the selected vehicle comes from.
 
 STR_CONFIG_SETTING_LANDSCAPE                                    :Landscape: {STRING2}
-STR_CONFIG_SETTING_LANDSCAPE_HELPTEXT                           :Landscapes define basic gameplay scenarios with different cargos and town growth requirements. NewGRF and Game Scripts allow finer control though
+STR_CONFIG_SETTING_LANDSCAPE_HELPTEXT                           :Landscapes define basic gameplay scenarios with different cargos and town growth requirements. NewGRF and game scripts allow finer control though
```

Comment:
GameScripts is a technical term and should be left (possibly with the space between words removed)

Reply:
Sure thing. You prefer Game Script then?

Reply:
I think so, but it's the sort of thing that might need arguing about with others :)

Reply:
Oops, I changed it all to Game Scripts already. Let me know, if you would like to have it changed again.